core/src/test/scala/com.mediative.sparrow/DataFrameReaderTest.scala

```diff
@@ -168,58 +153,12 @@ class DataFrameReaderTest extends FreeSpec with BeforeAndAfterAll {
     }
 
     def testFailure[T: RowConverter: ClassTag](json: Array[String], expected: NonEmptyList[String]) = {
+      val sqlContext = new SQLContext(sc)
       val df = sqlContext.jsonRDD(sc.parallelize(json))
 
       assert(toRDD[T](df) == expected.failure)
     }
 
-    // To get DataFrame#toRDD usage.
-    import com.mediative.sparrow.syntax.df._
-
-    "round-trip an object containing an Int and a String from RDD to DataFrame back to RDD" in {
```

Comment:
Thinking some more on this, in the future it might be better to categorize
the limitations based on the different components. We're not there yet and
I think the current limitations could probably be top level, but we may
need further breakdown within limitations later (which we can deal with in
the future when need arises).

Still +1 for having a limitations spec.

On Sun, Jul 26, 2015, 10:52 Jonas Fonseca <notifications@github.com> wrote:

> In core/src/test/scala/com.mediative.sparrow/DataFrameReaderTest.scala
> <https://github.com/ypg-data/sparrow/pull/18#discussion_r35493697>:
>
> >        val df = sqlContext.jsonRDD(sc.parallelize(json))
> >
> >        assert(toRDD[T](df) == expected.failure)
> >      }
> >
> > -    // To get DataFrame#toRDD usage.
> > -    import com.mediative.sparrow.syntax.df._
> > -
> > -    "round-trip an object containing an Int and a String from RDD to DataFrame back to RDD" in {
>
> I suggest we move pending tests to a separate "spec" called
> LimitationsSpec, where each case should document why it is there and
> refer to any tickets that are planned to resolve them.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ypg-data/sparrow/pull/18/files#r35493697>.
>